src/components/MyRecipes.vue

```diff
@@ -181,11 +230,24 @@
         </div>
     </div>
 </template>
-
+  
 <script>
-import UploadImage from './UploadImage.vue';
+import UploadImage from "./UploadImage.vue";
 import { VueEditor } from "vue2-editor";
-import FlagPicker from './Widgets/FlagPickerWidget.vue'
+import FlagPicker from './Widgets/FlagPickerWidget.vue';
+import { ValidationProvider, ValidationObserver } from 'vee-validate';
+import { extend } from 'vee-validate';
+import * as rules from 'vee-validate/dist/rules';
+import { messages } from 'vee-validate/dist/locale/en.json';
+import CookbookSelector from './Widgets/CookbookSelectorWidget';
+import { mapActions } from 'vuex';
+
+Object.keys(rules).forEach(rule => {
```

Comment:
Isn't there a better way to implement this?

Reply:
This is the initial code, I tried to change the code as little as possible